### PR TITLE
Require server opt-in with transport parameters

### PR DESCRIPTION
**Pull request by pthatcherg**
- Send web client transport parameter web_client 0x333C
- Check that the client's origin is in the server's transport parameter web_accepted_origins 0x333A (or the transport parameter is "*").

An attempt at solving https://github.com/w3c/webrtc-quic/issues/97 with Victor's idea.

### CODE REVIEW

**Comment by pthatcherg** [comment]
Note from Editor's call: seems like a good approach; ask Victor to review.

File: cs.html [hunk 1]
```diff
@@ -752,29 +752,43 @@ <h2>Constructors</h2>
...
+                    internal slot to <code>"failed"</code> and abort these steps.
+                </li>
+                <li>Let <var>joinedAcceptedOrigins</var> be the QUIC transport parameter provided
+                    by the server with ID web_accepted_origins(0x333A).  If the transport parameter is
+                    absent, set <var>quictransport</var>'s <a>[[\QuicTransportState]]</a>
+                    internal slot to <code>"failed"</code> and abort these steps.
+                    <!-- TODO: register 0x333A with IANA -->
+                </li>
+                <li>
+                    Let <var>serialiedAcceptedOrigins</var> be
```
Comment (shampson) [inline cs.html]: serialiedAcceptedOrigins --> "serializedAcceptedOrigins"

**Comment by pthatcherg** [comment]
Note from editor's meeting: waiting on Victor's review.  Ping him :)

File: cs.html (hunk 1 above)
Comment (pthatcherg) [reply cs.html]: Fixed.  Thanks for catching

**Comment by vasilvv** [comment]
This looks good to me.

**Pull request merged**